### PR TITLE
Removes xeno tips, a few other things

### PR DESCRIPTION
**Pull request by private-tristan**
# About the pull request
 removes xenotips, tips that mention a few things that dont exist, such as:

>defibs
IOs
 Intel
loaded OBs
 CPRing dead marines to extend death timer
 req (specifically, kept in supply drops due to admins sending those)
spec tokens
spec weapons
mortar
mentions of CIC staff being replaced with PLTCO

and edits a few others to remove mentions of xenos, along with adding Bravo, Charlie, and Delta to the lists of "squads that don't exist" 

probably missed a few.

# Explain why it's good for the game

xeno tips in PvE game where you are marines, mentions to things that don't exist.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
del: Xeno tips are gone
spellcheck: changed a few tips to remove mentions to things that don't exist (defibs, CPR, IOs, Intel)
add: added a few of meme tips.
/:cl:

### CODE REVIEW

File: strings/marinetips.txt [hunk 1]
```diff
@@ -7,21 +7,20 @@ Vary your loadout! Doing the same thing every round gets old fast. Try out the p
 Be careful when firing an Orbital Bombardment, Close Air Support, or the Mortar. Make sure you're not about to blow up the entire marine push.
 The Orbital Bombardment is always closer than it looks.
 Never inject a marine you haven't scanned! You might overdose them; ESPECIALLY so with an emergency autoinjector.
-Check perma-dead marines (skull icon on medhud) for useful gear, such as webbings, AP ammo, or binoculars.
+Check dead marines (skull icon on medhud) for useful gear, such as webbings, AP ammo, or binoculars.
 Always check for traps beneath loose objects - it could save your life!
 Bring a fire extinguisher, it might just save someone's life. They can remove flames that block a push, remove acid from other marines, as well as put out people on fire.
-Trading your life for a Xenomorph's is almost always worth it.
+Trading your life for a Xenomorph's is almost always never it.
```
Comment (morrowwolf) [inline strings/marinetips.txt:13]: never worth it

File: strings/marinetips.txt [hunk 2]
```diff
@@ -7,21 +7,20 @@ Vary your loadout! Doing the same thing every round gets old fast. Try out the p
...
 Try not to block off pushes when using a flamethrower.
-Recover bodies whenever possible. Even if the marine is unrevivable, useful gear and ammunition can still be salvaged.
+Recover bodies whenever possible. Useful gear and ammunition can be salvaged from their corpse.
 A lack of supplies at the FOB can cost you the round. Make sure to unpack supply crates, weapons, and ammo for your fellow marines.
 More dakka is not good when it is directed towards your teammates.
 Never place barricades facing each other - Not only does this block bullets, both barricades are damaged at once if one is slashed!
 An FOB is as strong as its weakest part. Secure all flanks, remember that walls can be pried open.
 Flamethrowers: Green fire is weak but has a lot of utility, such as melting through armor and being instantly extinguishable. Queens and Ravagers are flameproof. Most flamethrower damage comes from the initial burst, not the flame on the ground. Don't forget you have an underbarrel extinguisher.
-CPR extends the defib timer on dead marines. It can be done by clicking the dead marine on help intent with an open hand. Make sure to get the rhythm correct!
```
Comment (morrowwolf) [inline strings/marinetips.txt:24]: Probably just change this to talk about it counteracting oxyloss

File: strings/marinetips.txt [hunk 3]
```diff
@@ -35,29 +34,19 @@ Dragging a Nanomed onto yourself instantly gives a Health Analyzer report. Alway
...
-Researchers can make pressurized canister pouches, which can contain up to 480 reagents, and contain custom injectors that can be easily used by privates.
 When fighting xenomorphs, don't aim for the head - It won't increase damage against anyone but your fellow marines.
-Don't forget to set the Anti-Aircraft Gun!
 Scavenging attachments (or other things) from dead marines isn't bad practice, as long as you make sure they're actually dead first.
-When receiving a supply crate drop from requisitions, make sure to move the crate before opening it - Otherwise subsequent supply crates will be hidden beneath it!
+When receiving a supply crate drop, make sure to move the crate before opening it - Otherwise subsequent supply crates will be hidden beneath it!
 If a grenade comes your way, hit the deck! Lying down will reduce damage and let you dodge any shrapnel flying overhead.
 You can grab items from second-level storage, such as a splint in a first aid kit stored inside a satchel.
-By right clicking your medical belt and selecting "toggle belt mode", you can take pills out of bottles by simply clicking them once.
```
Comment (morrowwolf) [inline strings/marinetips.txt:45]: Still a good tip for here

File: strings/marinetips.txt [hunk 4]
```diff
@@ -35,29 +34,19 @@ Dragging a Nanomed onto yourself instantly gives a Health Analyzer report. Alway
...
 You can use a hand labeler (as found in squad prep rooms) to name your equipment and make it less likely to be stolen.
 You can use a health analyzer in hand (Z key) to check the last scan readout from it.
 Holocards are a useful triage tool for doctors and medics. Ensure you assign them (examine the marine with shift-click and select an appropriate holocard) to marines who have taken damage that cannot be healed without surgery. (Hint : major organ damage or larval infection = red card!)
-Escape pods are designed for only three occupants - more than that, or if a larger xenomorph is in the pod, and it will malfunction and blow up on launch.
-A misloaded OB can deviate severely from the intended target area - ensure you load them correctly!
-The XO and CO are trained in Power Loader use and engineering, and can load the OB.
-You can change what your SL tracker beacon is tracking by right clicking on your headset and clicking "Switch Tracker Target".
-Boilers emit light - not every glow from around the corner is friendly!
+You can change what your TL tracker beacon is tracking by right clicking on your headset and clicking "Switch Tracker Target".
```
Comment (morrowwolf) [inline strings/marinetips.txt:46]: SL still works here, we don't really have team leads as everything has been renamed

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: strings/marinetips.txt (hunk 3 above)
Comment (private-tristan) [reply strings/marinetips.txt:45]: actually as of [PR](https://github.com/cmss13-devs/cmss13/pull/4621) 4621 this tip isn't particularly relevant as all belts are set to this mode automatically.

File: strings/marinetips.txt (hunk 4 above)
Comment (private-tristan) [reply strings/marinetips.txt:46]: fair, i'll change it.

File: strings/marinetips.txt (hunk 1 above)
Comment (private-tristan) [reply strings/marinetips.txt:13]: fair

File: strings/marinetips.txt (hunk 2 above)
Comment (private-tristan) [reply strings/marinetips.txt:24]: fair.

**Review by morrowwolf — APPROVED** [review]

**Pull request merged**